week1/index.js

```diff
@@ -1,10 +1,71 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
-var http = require('http');
+// Declare global variables
+const http = require('http');
+//start server 
+const server = http.createServer();
+// the state which will be increased or decreased on the user request
+let state = 10; 
+//This port was already chosen by the assignment maker.
+const port = 8080;
+//Object containing the respons text and the content type.
+let responsToSendBack = {
+    code:'<html></html>',
+    type:'text/html',
+    statusCode: 200
+};
 
-var port = 8080;
 
-var server = http.createServer();
+// Function that checks which URL input we got from the user 
+//Initiates the right action based on the user choice
+function checkInput (url) {
+    switch (url) {
+        case '/state':
+            stateResponsHtml();
+            break;
+        case '/add':
+            addOne();
+            operationsResponseHtml();
+            console.log(state);
+            break;
+        case '/remove':
+            subOne();
+            operationsResponseHtml();
+            console.log(state);
+            break;
+        case '/reset':
+            state = 10;
+            operationsResponseHtml();
+            break;
+        default:
+            otherUrls();
+    }
+}
+
+//Function to increase the state by 1
+function addOne (){
+    return state++;
+};
+//Function to decrease the state
+function subOne () {
+    return state--;
+};
+
+//Function to build the response HTML for subtraction and addition
```

Comment:
I love that you were verbose and clear. I did notice that your use of comments is effectively 'WHAT' the function does (which I can usually figure out from the function name), and not 'WHY' your functions do things in a certain way. It's worth spending a few extra lines explaining things clearly.

For example, in this case, I'd write something like:
```
// This function builds the HTML response for endpoints that change the value.
// Since the value doesn't need to be printed to acknowledge a change, this response
//    simply returns an acknowledgement without any content.
```

Good job on reading the assignment clearly, and just sending back the HTML with 'ok'. I didn't notice this until now, and you'll see when I share my example of the homework, I didn't follow the instructions clearly enough here 😬